backend/compact-connect/stacks/api_stack/v1_api/api_model.py

```diff
@@ -0,0 +1,493 @@
+                            'ssn': JsonSchema(
+                                type=JsonSchemaType.STRING,
+                                description='Social security number to look up',
+                                pattern=cc_api.SSN_FORMAT
+                            ),
+                            'providerId': JsonSchema(
+                                type=JsonSchemaType.STRING,
+                                description='Internal UUID for the provider',
+                                pattern=cc_api.UUID4_FORMAT
+                            ),
+                            'jurisdiction': JsonSchema(
+                                type=JsonSchemaType.STRING,
+                                description="Filter for providers with privilege/license in a jurisdiction",
+                                enum=self.api.node.get_context('jurisdictions')
+                            )
+                        }
+                    ),
+                    'pagination': self._pagination_request_schema,
+                    'sorting': self._sorting_schema
+                }
+            )
+        )
+        return self.api._v1_query_providers_request_model
+
+    @property
+    def query_providers_response_model(self) -> Model:
+        """
+        Return the query providers response model, which should only be created once per API
+        """
+        if hasattr(self.api, '_v1_query_providers_response_model'):
+            return self.api._v1_query_providers_response_model
+        self.api._v1_query_providers_response_model = self.api.add_model(
+            'V1QueryProvidersResponseModel',
+            description='Query providers response model',
+            schema=JsonSchema(
+                type=JsonSchemaType.OBJECT,
+                required=['items', 'pagination'],
+                properties={
+                    'providers': JsonSchema(
+                        type=JsonSchemaType.ARRAY,
+                        max_length=100,
+                        items=self.providers_response_schema
+                    ),
+                    'pagination': self._pagination_response_schema,
+                    'sorting': self._sorting_schema
+                }
+            )
+        )
+        return self.api._v1_query_providers_response_model
+
+    @property
+    def provider_response_model(self) -> Model:
+        """
+        Return the provider response model, which should only be created once per API
+        """
+        if hasattr(self.api, '_v1_get_provider_response_model'):
+            return self.api._v1_get_provider_response_model
+        self.api._v1_get_provider_response_model = self.api.add_model(
+            'V1GetProviderResponseModel',
+            description='Get provider response model',
+            schema=self._v1_provider_detail_response_schema
+        )
+        return self.api._v1_get_provider_response_model
+
+    @property
+    def providers_response_schema(self):
```

Comment:
Looks like this property can be made internal, now (and moved below with the other internals).